generator/.DevConfigs/3436963d-3bfd-4d4b-a929-bc652e5c1b4d.json

```diff
@@ -0,0 +1,11 @@
+{
+    "services": [
+        {
```

Comment:
You also changed `Core` (`sdk/src/Core/Amazon.Util/AWSSDKUtils.cs`) but didn't include it here.

Reply:
Thanks for catching it. Fixed.